spec/system/ocr_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'OCR', :paperclip_processing, :sidekiq_inline do
+  include ProfileStories
+
+  subject { page }
+
+  let(:email)               { 'test@example.com' }
+  let(:password)            { 'password' }
+  let(:confirmed_at)        { Time.zone.now }
+  let(:finished_onboarding) { true }
+
+  before do
+    as_a_logged_in_user
+    visit root_path
+  end
+
+  it 'can recognize text in a media attachment' do
+    expect(subject).to have_css('div.app-holder')
+
+    within('.compose-form') do
+      attach_file('file-upload-input', Rails.root.join('spec', 'fixtures', 'files', 'text.png'), make_visible: true)
```

Comment:
The `fixture_file_upload` helper might be good shortcut here to get the file.

Reply:
Afaik the `fixture_file_upload` helper isn't usable in this situation, as `attach_file` expects a path.

Reply:
Sorry, good point - we're in capybara world here. That said, I think rspec provides an `file_fixture` which might shortcut you similarly through that dir path.